dashboards/ruby_vm/gvl.json

```
@@ -0,0 +1,62 @@
+{
+  "metric_keys": [
+    "gvl_global_timer",
+    "gvl_waiting_threads"
+  ],
+  "dashboard": {
+    "title": "Ruby GVL",
+    "description": "Metrics for the Ruby interpreter's global VM lock (GVL)",
+    "visuals": [
+      {
+        "title": "Global timer",
+        "description": "The time spent waiting to acquire the lock and resume execution, for all threads",
+        "line_label": "%name%",
+        "display": "LINE",
+        "format": "duration",
+        "draw_null_as_zero": true,
+        "metrics": [
+          {
+            "name": "gvl_global_timer",
+            "fields": [
+              {
+                "field": "GAUGE"
+              }
+            ],
+            "tags": [
+              {
+                "key": "hostname",
+                "value": "*"
+              }
+            ]
+          }
+        ],
+        "type": "timeseries"
+      },
+      {
+        "title": "Waiting threads",
+        "description": "The number of threads waiting to be resumed, at a given point in time",
```

```suggestion
        "description": "The number of threads waiting to be resumed",
```
That may be clear from the context of the time based graph.

---

Yeah, it's not the clearest wording, I'm happy to remove it. 😅

I've used [similar wording before](https://github.com/appsignal/public_config/blob/main/dashboards/nginx/main.json#L171), to attempt to make it clear that, unlike other metrics, it's not cumulative over the time between measurements. I don't think it's too ambiguous in this case, as the cumulative reading for this metric wouldn't make a lot of sense.

---

ah I missed that. But the part where it talks about "minutes" would also not work in the hourly metric view: 24 hour view and up.